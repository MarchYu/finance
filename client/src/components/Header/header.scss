.header {
    height: 8%;
    width: 100%;
    background-color:#1D2D44;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    font-size: 1.2em;
    padding: 20px;
    box-sizing: border-box;
    color:aliceblue;
}