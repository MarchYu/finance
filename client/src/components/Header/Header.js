import "./header.scss";

export const Header = () => {
  return <header className="header">Hover for change percent</header>;
};
