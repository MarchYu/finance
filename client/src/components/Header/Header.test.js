import { screen, render } from "@testing-library/react";
import { Header } from "./Header";

test("renders header logo", () => {
  render(<Header />);
  const HeaderLogoElement = screen.getByText(/Hover for change percent/i);
  expect(HeaderLogoElement).toBeInTheDocument();
  expect(HeaderLogoElement).toMatchSnapshot();
  expect(HeaderLogoElement).toHaveClass("header");
});
