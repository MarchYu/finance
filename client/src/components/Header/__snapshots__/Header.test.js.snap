// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header logo 1`] = `
<header
  class="header"
>
  Hover for change percent
</header>
`;
